CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ##### Enhancements
 
-* None.
+* By default, docs are now searchable. A new option, `--unsearchable`, lets you
```

Comment:
Only the `fullwidth` theme appears to be searchable. Also, this should be reformatted to match the standard changelog entry format defined in https://github.com/realm/jazzy/blob/master/CONTRIBUTING.md#tracking-changes

With credit given to @esad, @tmcw and @captainbarbosa